### DISCUSSION TITLE
Distance between claims

### DISCUSSION THREAD

**Discussion by UmutBey12122** [GriefPrevention/GriefPrevention#1867] [Q&A]
Hello, I  want players to be at least 100 blocks apart when making claims. How can I do?

**Comment by bigpresh** [comment]
Have them make their claims bigger if they want a bit of space between them and potential neighbours :)

This has been discussed in the past, and the potential for confusion with odd "no-mans-land" between claims and the implementation of such was decided not a good idea.  See e.g. #124 and #505 

It could be potentially be done by an add-on, if someone wanted to write one.
